normative/dtpr-stickers
# Changes feedback questions and add expanded survey config link

# Notion card link(s)
- [Change Feedback Questions and Add Survey Link](https://www.notion.so/normative/Change-Feedback-Questions-and-Add-Survey-Link-647952d6394e4870808552a0cfc5e18d)

# Review APP
- [Heroku Review App](https://dtpr-sticker-feature-fe-9bcbsb.herokuapp.com/sensors/slcJ3Jk93uilfXcZaOqhG)

# PR Description (Proposed changes, approach, gotchas, etc)
- [x] Adds environment variable for expanded survey link
- [x] Changes feedback questions

# How to test it
- Access Review App through the link on the top of description and check feedback questions
- Check if feedback questions follow what was described into [Notion Card](https://www.notion.so/normative/Change-Feedback-Questions-and-Add-Survey-Link-647952d6394e4870808552a0cfc5e18d)
